conda/activate.py

```diff
@@ -651,6 +668,18 @@ def _get_deactivate_scripts(self, prefix):
             prefix, 'etc', 'conda', 'deactivate.d', '*' + self.script_extension
         )), reverse=True))
 
+    def _get_environment_env_vars(self, prefix):
+        env_vars = {}
+        env_vars_file = join(prefix, 'etc', 'conda', 'env_vars')
```

Comment:
I think this needs to go in `conda-meta/`. That’s the location that’s completely off limits to packages. Otherwise any rando package on anaconda.org could overwrite that location.

Reply:
good call!